src/ory-editor/converter/src/createPlugin.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Created by benny on 17.11.16.
+ */
+import { v4 } from 'uuid'
+import markdownToSlate from './markdownToSlate'
+
+const createPlugins = ({ normalized, elements }) => {
+  const split = normalized
+    .split(/(§\d+§)/)
+    .map(s => s.trim())
+    .filter(s => s !== '')
+  return split.map(markdown => {
+    // console.log(markdown)
+    var elementID = /§(\d+)§/.exec(markdown)
+    // console.log(elementID)
+    if (elementID !== null) {
+      return {
+        cells: [createPluginCell(elements[elementID[1]])]
+      }
+    } else {
+      return {
+        cells: [
+          {
+            markdown: markdown
+          }
+        ]
+      }
+    }
+  })
+}
+const createPluginCell = elem => {
+  switch (elem.name) {
+    case 'table':
+      return {
+        content: {
+          plugin: {
+            name: 'serlo/content/markdown'
```

Comment:
@Knorrke: couldn't find the plugin... ^^'

Reply:
Ah this should be `serlo/content/markdown-table`, I renamed that -.- it's in the ory-editor-plugins repo on branch markdown with open PR to master.